harlog.go

```diff
@@ -124,10 +135,23 @@ func (ctx *ProxyCtx) LogToHARFile(captureContent bool) Next {
 	return NEXT
 }
 
+func (ctx *ProxyCtx) GetHar(filename string) {
+	ctx.proxy.FlushHARToDisk(filename)
+}
+
 func (ctx *ProxyCtx) FlushHARToDisk(filename string) {
 	ctx.proxy.FlushHARToDisk(filename)
 }
 
+func (proxy *ProxyHttpServer) FlushHARMemory() (chan *har.Har){
+	proxy.Logf("Calling a flush of HAR to memory")
+	harMemory := make(chan *har.Har)
+
+	proxy.HarFlushToMemory <- harMemory
```

Comment:
all of this seems pretty convoluted to me.. what exactly are you trying to achieve ? :)

I see there's a potential feature we could merge in here.. just not sure if this is the way to do it :)